virt/virt-4-9-release-notes.adoc

```diff
@@ -46,6 +46,9 @@ Red Hat is committed to replacing problematic language in our code, documentatio
 
 //CNV-9540 High performance windows
 
+//CNV-8875
+* If your {VirtProductName} Operator subscription used any update channel other than *stable*, it is now automatically subscribed to the *stable* channel. The *stable* channel subscribes you to minor version updates and ensures that your {VirtProductName} version is compatible with your {product-title} version.
```

Comment:
Can you please change {VirtProductName} and {product-title} to bold text .?

Reply:
@sbulage Per our style guide, we use bold text when referring to a specific UI item, and not to highlight or emphasize.